src/main/java/at/fhv/td/domain/interfaces/IArtist.java

```diff
@@ -11,5 +11,5 @@
 
     Set<Tour> getTours();
 
-    Long getArtistId();
+    Long getId();
```

Comment:
Wie wärs, wenn wir das `getId` aus dem Interface raus lassen und stattdessen in den Domainobject interfaces das `IModelId` interface inplementieren -- dann in den Domainobjekten selbst nicht mehr?

So ist `getId` nämlich doppelt denke ich. 🙂 
(Nur ein Vorschalg)

Reply:
find ich nicht gut, da im domain interface nur getter sind... und wir benötigen ein einheitliches setId und damit kann ein DomainInterface nicht von dem IModelId Interface erben.
Das mit dem doppelt verstehe ich gut, das find ich auch nicht toll, allerdings besser als ein setter im DomainInterface 😉

Reply:
Ok hast recht. 👌